catalog/ui/catalog-ui-search/src/main/webapp/component/navigation-left/navigation-left.less

```diff
@@ -1,10 +1,18 @@
 @{customElementNamespace}navigation-left {
     .customElement;
     position: relative;
+    overflow: hidden;
+    cursor: pointer;
 
     button {
         text-align: center;
         width: @minimumButtonSize;
         height: 100%;
     }
+
+    button > .navigation-indicator {
+        position: absolute;
+        right: 0px;
+        top: -5px;
```

Comment:
Should this be sized in rem so when you resize page the indicator stays where it is relative to the text its next to?